composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ayeo/gs1_barcode",
+  "name": "matias-vdv/gs1_barcode",

```

Comment:
This should not be part of PR ;)

Reply:
Absolutely! I'm sorry, first time i've done this lol ;). 
Made the pull request before i did this change since i needed it fast and i didn't expect you to respond this fast. And for some reason i thought this wouldn't be added to the pull request, stupid me :D
I removed it

Reply:
I am not an expert as well here ;) Just asking.